src/components/SignupForm/index.js

```diff
@@ -85,12 +85,12 @@ const SignupForm = () => {
         light
         small
         css={css`
-          max-width: 48rem;
+          max-width: 56rem;
+          font-size: 1.6rem;
         `}
       >
-        Your information will only be used to send beta notifications, and will never be sold to
-        anyone. You can unsubscribe at any time. Beta users will receive discounts on the full
-        product and our undying affection.
+        Your information will only be used to send beta notifications.{' '}
+        <Link to="/privacy">Privacy.</Link>
```

Comment:
This single-word sentence strikes me as odd. What about:

```suggestion
        <Link to="/privacy"><VisuallyHidden>Read out </VisuallyHidden>Privacy<VisuallyHidden> Policy</VisuallyHidden></Link>
```